daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -666,6 +671,14 @@ optionsParser numProcessors parsePkgName = Compiler.Options
         metavar "OPTION" <>
         help "Options to pass to the underlying GHC"
 
+    optScenarioService :: Parser Bool
+    optScenarioService = fmap not .
+        switch $
+            help "Disable scenario service." <>
+            long "disable-scenario-service" <>
```

Comment:
Given that we tried to avoid these slightly confusing negative flags in other places with `flagYesNoAuto` should we do this here as well?

Reply:
Done! I added an extra argument in flagYesNoAuto so I can make the `scenario-service` flag internal.